src/GameLogic/ItemPriceCalculator.cs

```diff
@@ -438,38 +573,6 @@ private static long CalculateBuyingPrice(Item item, byte durability)
             price = MaximumPrice;
         }
 
-        var maxDurability = item.GetMaximumDurabilityOfOnePiece();
-        if (maxDurability > 1 && maxDurability > durability)
-        {
-            float multiplier = 1.0f - ((float)durability / maxDurability);
-            long loss = (long)(price * 0.6 * multiplier);
-            price -= loss;
-        }
-
-        return RoundPrice(price);
```

Comment:
We can't round here yet because for the selling price dividing by 3 comes first:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/common/zzzitem.cpp#L2634